mw-webapp/src/component/header/Header.tsx

```diff
@@ -28,7 +28,7 @@ export const Header = () => {
       </h1>
       <div className={styles.block_button}>
```

Comment:
rename classname . we don't use _ for classes. Use camelCase

Reply:
Done